qt/python/mantidqt/mantidqt/widgets/codeeditor/multifileinterpreter.py

```diff
@@ -347,12 +379,20 @@ def restore_session_tabs(self):
 
     def save_current_file(self):
         """Save the current file"""
-        self.current_editor().save(force_save=True)
+        self.monitor_files_changing = False
+        try:
+            self.current_editor().save(force_save=True)
+        finally:
+            self.monitor_files_changing = True
```

Comment:
Yes, I wanted to make sure that if save() goes wrong then the flag gets left in the correct state, otherwise the file monitoring will be broken until Workbench is restarted.